styles/main.css

```diff
@@ -7,6 +7,8 @@
 @import url("../styles/button.css");
 @import url("../styles/card.css");
 @import url("../styles/dialogue.css");
+@import url("../styles/image.css");
```

Comment:
though adding imports like this works but it is very inefficient when it comes to CSS. I see 10+ network requests to get all CSS and the page has a very slow load time as a result.